src/org/feedhenry/Utils.groovy

```diff
@@ -36,3 +38,9 @@ def getArtifactsDir(name) {
 def gitRepoIsDirty(untrackedFiles='no') {
     return sh(returnStdout: true, script: "git status --porcelain --untracked-files=${untrackedFiles}").trim()
 }
+
+static Closure getDate() {
+    Date now = new Date()
+    SimpleDateFormat yearMonthDateHourMin = new SimpleDateFormat("yyyyMMddHHmm")
```

Comment:
I've changed this from the format in the wendy build jobs, to include the hour and minute, in case we want to do more than one build on the same day (the output of this is used as a Docker tag in a BuildConfig)